projects/prod_test/PowerI2CCommands.c

```diff
@@ -46,6 +48,24 @@ struct dev_i2c_addr_t pm_addrs_dcdc[N_PM_ADDRS_DCDC] = {
      POWER_I2C_F2AVTTVCC_ADDR},
 };
 
+// poweron check constants
```

Comment:
maybe add a comment here what the four fields of the struct are, or use designated initializer to initialize to make it more obvious